src/content/changelog/4-62-0-2020-04-10.md

```diff
@@ -16,14 +16,16 @@ changelog:
 
 - The View Kubernetes API Endpoint ([/lke/clusters/{clusterId}/api-endpoint](/api/v4/lke-clusters-cluster-id-api-endpoint)) endpoint is now deprecated in favor of List Kubernetes API Endpoints ([GET /lke/clusters/{clusterId}/api-endpoints](/api/v4/lke-clusters-cluster-id-api-endpoints)).
 
-- The `version` property of the LKE collection has been updated to be named `k8s_version`. This change affects the following endpoints:
+- The `version` property of an LKE cluster has been updated to be named `k8s_version`. This change affects the following endpoints:
```

Comment:
Wanted to clarify this a bit, as the LKE collection also has two endpoints for viewing/listing kubernetes versions, and those are unaffected.